lib/plugins/aws/invokeLocal/index.test.js

```diff
@@ -1,1362 +1,1686 @@
-      });
-    });
-
-    it('it should set credential env vars #2', () => {
-      provider.cachedCredentials = {
-        credentials: {
-          sessionToken: 'TOKEN',
-        },
-      };
-      return awsInvokeLocal
-        .loadEnvVars()
-
-        .then(() => {
-          expect(process.env.AWS_SESSION_TOKEN).to.equal('TOKEN');
-          expect('AWS_ACCESS_KEY_ID' in process.env).to.equal(false);
-          expect('AWS_SECRET_ACCESS_KEY' in process.env).to.equal(false);
-        });
-    });
-
-    it('it should work without cached credentials set', () => {
-      provider.cachedCredentials = null;
-      return awsInvokeLocal
-        .loadEnvVars()
-
-        .then(() => {
-          expect('AWS_SESSION_TOKEN' in process.env).to.equal(false);
-          expect('AWS_ACCESS_KEY_ID' in process.env).to.equal(false);
-          expect('AWS_SECRET_ACCESS_KEY' in process.env).to.equal(false);
-        });
-    });
-
-    it('should fallback to service provider configuration when options are not available', () => {
-      awsInvokeLocal.provider.options.region = null;
-      awsInvokeLocal.serverless.service.provider.region = 'us-west-1';
-
-      return awsInvokeLocal.loadEnvVars().then(() => {
-        expect(process.env.AWS_REGION).to.equal('us-west-1');
-        expect(process.env.AWS_DEFAULT_REGION).to.equal('us-west-1');
-      });
-    });
-
-    it('it should overwrite provider env vars', () => {
-      awsInvokeLocal.options.functionObj.environment.providerVar = 'providerValueOverwritten';
-
-      return awsInvokeLocal.loadEnvVars().then(() => {
-        expect(process.env.providerVar).to.be.equal('providerValueOverwritten');
-      });
-    });
-  });
-
-  describe('#invokeLocal()', () => {
-    let invokeLocalNodeJsStub;
-    let invokeLocalPythonStub;
-    let invokeLocalJavaStub;
-    let invokeLocalRubyStub;
-    let invokeLocalDockerStub;
-
-    beforeEach(() => {
-      invokeLocalNodeJsStub = sinon.stub(awsInvokeLocal, 'invokeLocalNodeJs').resolves();
-      invokeLocalPythonStub = sinon.stub(awsInvokeLocal, 'invokeLocalPython').resolves();
-      invokeLocalJavaStub = sinon.stub(awsInvokeLocal, 'invokeLocalJava').resolves();
-      invokeLocalRubyStub = sinon.stub(awsInvokeLocal, 'invokeLocalRuby').resolves();
-      invokeLocalDockerStub = sinon.stub(awsInvokeLocal, 'invokeLocalDocker').resolves();
-
-      awsInvokeLocal.serverless.service.service = 'new-service';
-      awsInvokeLocal.provider.options.stage = 'dev';
-      awsInvokeLocal.options = {
-        function: 'first',
-        functionObj: {
-          handler: 'handler.hello',
-          name: 'hello',
-        },
-        data: {},
-      };
-    });
-
-    afterEach(() => {
-      awsInvokeLocal.invokeLocalNodeJs.restore();
-      awsInvokeLocal.invokeLocalPython.restore();
-      awsInvokeLocal.invokeLocalJava.restore();
-      awsInvokeLocal.invokeLocalRuby.restore();
-    });
+        expect(testResult).to.be.fulfilled.then(({stdoutData}) => {
```

Comment:
`testResult` is already fulfilled, note it's a resolved value (you have `testResult = await runServerless`)

So maybe issue is, that you've used here a promise testing interface for value that's not a promise?